src/pyk/__main__.py

```diff
@@ -147,7 +152,8 @@ def create_argument_parser() -> ArgumentParser:
         help='Pretty print a term.',
         parents=[k_cli_args.logging_args, definition_args, k_cli_args.display_args],
     )
-    print_args.add_argument('term', type=FileType('r'), help='Input term (in JSON).')
+    print_args.add_argument('term', type=FileType('r'), help='Input term (in format specified with --input).')
+    print_args.add_argument('--input', default='kast', type=str, help='')
```

Comment:
This shoul dbe done with an enum of `PrintInput` types. See `PrintOutput` for example: https://github.com/runtimeverification/pyk/blob/f0f2e0f3368edd032b1cbe83b74e02dcd1fe0e8b/src/pyk/kore/tools.py#L12.

```
class PrintInput(Enum):
    KOREJSON = 'kore-json'
    KASTJSON = 'kast-json'
```

```suggestion
    input_formats = '|'.join([pi.value for pi in PrintInput])
    print_args.add_argument('--input', default='kast', type=PrintInput, help=f'Format of input: [{input_formats}]')
```

Reply:
Done